.github/workflows/buildsCI.yaml

```diff
@@ -30,10 +30,16 @@ jobs:
       run: |
        New-ItemProperty -Path "HKLM:\SYSTEM\CurrentControlSet\Control\FileSystem" -Name "LongPathsEnabled" -Value 1 -PropertyType DWORD -Force
     
+    - name: Check_API
+      shell: bash
+      run: |
+        cd scripts
+        ./api_check.sh
```

Comment:
So my question is:
Will the last `cd ..` still happen, even if api_check.sh fails?